mw-webapp/src/component/progressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,41 @@
+import styles from "src/component/progressBar/ProgressBar.module.scss";
+
+/**
+ * Type props
+ */
+interface ProgressBarProps {
+
+  /**
+   * Text to show in progress bar
+   * No text by default
+   */
+  text?: string;
+
+  /**
+   * Fill percentage
+   * (minimum 0 maximum 100)
+   */
+  percentage: number;
+}
+
+/**
+ * ProgressBar component
+ */
+export const ProgressBar = (props: ProgressBarProps) => {
+  if (!isNaN(props.percentage)) {
```

Comment:
Do we need this check?  Is it possible that props.percentage is not number?  IN interface percentage is only number

Reply:
Done